runbooks/how-to-create-server-regression-dashboards.md

```diff
@@ -0,0 +1,343 @@
+   3. Enter your AWS account credentials and sign-in
+   4. You should now have navigated to the "Console Home", click the search input in the navigation bar on the top left of the screen
+   5. Type "QuickSight" and click the QuickSight result that appears
+
+2. Every QuickSight dashboard requires a _Dataset_, which for the `server-regression` dashboards will be a `Custom SQL` query against Athena. As there will be three dashboards, there will be three datasets (one dataset for each dashboard)
+
+   1. Creating the `bfd-test-server-regression` dataset:
+
+      1. Navigate to "Datasets" in QuickSight (click on "Datasets" in the navigation tree on the left side of the page)
+      2. Click the "New dataset" button on the top-left of the page
+      3. The page that appears should list a large number of possible data sources. Find "Athena" in this list, and click on it
+      4. Enter "bfd-test-server-regression" for the "Data source name"
+      5. Choose `bfd` for the "Athena workgroup"
+      6. Click "Create data source"
+      7. Choose `AwsDataCatalog` as the "Catalog"
+      8. Choose `bfd-insights-bfd-test` as the "Database"
+      9. Choose `bfd_insights_bfd_test_server_regression" as the "Table"
+      10. Click "Use custom SQL"
+      11. In the first input box, enter "bfd-test-server-regression" (replacing "New custom SQL")
+      12. In the large text area, enter the following SQL query:
+
+      ```sql
+      select
+        totals.total_reqs_per_second,
+        totals.num_requests as total_requests,
+        totals.num_failures as total_failures,
+        totals.min_response_time as total_min_response_time,
+        totals.average_response_time as total_avg_response_time,
+        totals.median_response_time as total_median_response_time,
+        totals.response_time_percentiles."0.95" as total_95_percentile,
+        totals.response_time_percentiles."0.99" as total_99_percentile,
+        totals.response_time_percentiles."1.0" as total_100_percentile,
+        metadata.compare_result,
+        date_format(from_unixtime(metadata.timestamp, 'US/Eastern'), '%b-%e-%y %T EST') as date_time,
+        metadata.timestamp as raw_timestamp
+      from "bfd-insights-bfd-test"."bfd_insights_bfd_test_server_regression"
+      where
+        metadata.hash = 'ae770dea971f3be467a42717654a42c5c26c7da394f690c3db23e9aa4c4e2f92'
+        and contains(metadata.tags, 'master')
+        and (metadata.total_runtime - 30) < 3.0
+      order by metadata.timestamp asc
+      ```
+
+      13. Click "Confirm query"
+      14. In "Finish dataset creation", select "Directly query your data"
+      15. Click "Edit/preview data"
+      16. An editor should appear with the SQL you entered above, click "Apply"
+      17. Ensure that data is returned in the data preview
+      18. Click "SAVE & PUBLISH" in the navigation bar on the top right of the screen
+      19. Ensure that a toast appears indicating the dataset was saved successfully
```

Comment:
Yeah, a ["toast"](https://stackoverflow.com/questions/1348788/what-does-toast-mean) in this context would be analogous to an in-page notification (visible at the top-right of the screen).

I know "toast" was valid terminology for that particular kind of UI-element for front-end (and mobile, specifically Android's Material) _at some point_, but it might have fallen out of favor over the last few years. I'll replace "toast" with "in-page notification" so that it's clearer.